misc/add-netbird-lxc.sh

```diff
@@ -52,6 +54,23 @@ while [ -z "${CTID:+x}" ]; do
     "${CTID_MENU[@]}" 3>&1 1>&2 2>&3) || exit
 done
 
+LXC_STATUS=$(pct status "$CTID" | awk '{print $2}')
```

Comment:
This check is done like this in other scripts such as `clean-lxcs.sh`, `update-lxcs.sh`, `update-lxcs-cron.sh`
```
    status=$(pct status $container)
    template=$(pct config $container | grep -q "template:" && echo "true" || echo "false")
    if [ "$template" == "false" ] && [ "$status" == "status: stopped" ]; then
      echo -e "${BL}[Info]${GN} Starting${BL} $container ${CL} \n"
      pct start $container
      echo -e "${BL}[Info]${GN} Waiting For${BL} $container${CL}${GN} To Start ${CL} \n"
      sleep 5
      clean_container $container
      echo -e "${BL}[Info]${GN} Shutting down${BL} $container ${CL} \n"
      pct shutdown $container &
    elif [ "$status" == "status: running" ]; then
      clean_container $container
    fi
```

Perhaps there should be an `install_netbird` function so it can be implemented in a similar manner

Reply:
Then you have to completely revert the script again. It hardly adheres to any conventions anyway. I have rather less desire ^^ that's how it works in any case 😜

Reply:
I would atleast use this part instead of a `while` loop, since it can end up as a neverending loop if something goes wrong
```
echo -e "${BL}[Info]${GN} Waiting For${BL} $container${CL}${GN} To Start ${CL} \n"
sleep 5
```

Reply:
I have tested installing this block, but it doesn't fit at all. The changeover for this to work is too big for me to do it. The whole script and the process would have to be adapted. It works like this in any case, even on absolutely undersized servers. If you're tired, close the PR and customize it yourself :smile: